crates/tabby-common/src/api/structured_doc.rs

```diff
@@ -65,6 +66,15 @@ pub struct DocSearchIssueDocument {
     pub closed: bool,
 }
 
+#[derive(Clone)]
+pub struct DocSearchPullRequest {
+    pub title: String,
+    pub link: String,
+    pub body: String,
+    pub patch: String,
```

Comment:
Add document on this field is in patch / diff format (which standard, example, etc)